packages/Datepicker/src/Datepicker.tsx

```diff
@@ -271,19 +275,17 @@ const Datepicker: React.FunctionComponent<DatepickerProps> = ({
     }
 
     return (
-        <I18nProvider locale={locale}>
-            <Dropdown
-                isOpen={!disabled && isOpen}
-                onClose={onClose}
-                content={calendar}
-                size="medium"
-                className="ids-datepicker__dropdown"
-                dataTest={dataTest}
-                {...rest}
-            >
-                <Input {...inputProps} />
-            </Dropdown>
-        </I18nProvider>
+        <Dropdown
```

Comment:
hmmmmmmmmmmm are you 100% sure that removing the i18nProvider here is ok? 

Even if i know it's the same provider that is set in our context, the way igloo builds with rollup, they inline the code of 3rd party. So the version of our i18nProvider could be different from the one from the dropdown. 

i think it's safer here to keep it. 

Not going to be the case in hopper however

Reply:
but we want it to use our provider. That's why I added the calendar stuff to our provider.
I tested it and it was working.